metadata.yaml

```
@@ -0,0 +1,21 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: finos-waltz-k8s
+display-name: FINOS Waltz Server
+
+summary: |
+  Waltz allows you to visualize and define your organisation's technology landscape.
+
+description: |
+  In a nutshell, Waltz allows you to visualize and define your organisation's
+  technology landscape. Think of it like a structured Wiki for your architecture.
+
+containers:
+  waltz:
+    resource: waltz-image
+
+resources:
+  waltz-image:
+    type: oci-image
+    description: OCI image for waltz (ghcr.io/finos/waltz)
```

```suggestion
    description: OCI image for waltz (ghcr.io/finos/waltz)
    upstream-source: ghcr.io/finos/waltz
```

This might be a useful addition for when it comes to automating deployments as part of integration testing